docs/xxparallelcmsthreads.md

```diff
@@ -0,0 +1,37 @@
+<!--
+* Copyright (c) 2020, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:ParallelCMSThreads
+
+This Oracle Hotspot option affects the number of threads the concurrent garbage collector will use. This option is recognized by OpenJ9 and provided for compatibility.
```

Comment:
Please avoid future tense and rework similar to last comment.

Reply:
And HotSpot spelling too please.

Reply:
See https://github.com/eclipse/openj9-docs/pull/464/commits/094dc778d3f9c2c3f1e8f9758dfef579be8f1759